cmd/installer/user_all.go

```diff
@@ -15,8 +15,8 @@ import (
 	"syscall"
 )
 
-func moveToDDInstaller() error {
-	grp, err := user.LookupGroup("dd-installer")
+func moveToDDAgent() error {
+	grp, err := user.LookupGroup("dd-agent")
```

Comment:
dd-agent group is looked twice now instead of once, see below

Reply:
Good catch, fixed in [46e9146](https://github.com/DataDog/datadog-agent/pull/24919/commits/46e91465654e311c1c7835f504841dbd5f8c4526)